src/python/bot/fuzzers/ml/rnn/constants.py

```diff
@@ -53,10 +53,10 @@
 DISPLAY_LEN = 138
 
 # Training script name.
-TRAINING_SCRIPT_NAME = 'train.py'
+TRAINING_SCRIPT_NAME = 'ml.rnn.train'
```

Comment:
See why - https://stackoverflow.com/questions/16981921/relative-imports-in-python-3